src/main/markdown/release-notes.md

```diff
@@ -62,6 +63,63 @@ The GWT Release Notes
 * * *
 
 <a id="Release_Notes_Current"></a>
+## <a id="Release_Notes_2_9_0"></a> Release Notes for 2.9.0
+
+### Highlights
+- Able to compile projects with jsinterop-base 1.0.0, elemental2 1.0.0, and
+jsinterop-annotations 2.0.0. With the exception of @JsAsync and @JsEnum, this
+brings GWT2 to be compatible across these tools with J2CL.
+
+- Added support for Java language levels 9, 10, and 11.
+
+- Officially, support is dropped for running the GWT compiler or server-side
+tooling on Java 7. The GWT distribution is still compiled to run on Java 7
+for this release, but no guarantees are made about whether or not this will 
+work. Future versions will compile bytecode for Java 8+. The release was 
+tested and found to work cross platform when run with Java 8, 11, and 14.
+
+
+### Deprecations
+- Officially deprecated the original JSNI-based elemental - with elemental2
+reaching 1.0.0, it will take over for the old elemental library, which will
+not be compatible with J2CL/GWT3. To make this clear, all classes in elemental
+have been marked as deprecated - future releases of GWT may or may not include
+this library at all.
+- Removed NoSuchMethodException emulation
+
+### Bug fixes
+- Fix Arrays.binarySearch semantics for float[] and double[]
+- Method references should correctly implement their intersection cast
```

Comment:
Reword this one. Are these directly scrapped from cl descriptions?

Reply:
Yes, that's the standard approach since there are so many, and the subject line serves as a concise summary of the change which someone can use for searching quickly (git history being indexed by search engines, etc).

Reply:
If they are scrapped directly better to prepend the commit id, that way you could leave the description verbatim (sometime the first line of the description is not too informative nor gramatically correct, but hey). Otherwise each line should be rewritten to make sense to the reader.